test_themes/tests/__init__.py

```diff
@@ -1,7 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
Just a question (I am not good with all the modules and python stuff :sweat_smile:): at the end of the problematic test, the buzzy module is reinstalled right ? So shouldn't the `test-themes` one have been reinstalled with it too ? (I am not sure on how the dependencies work)
If not, then why not reinstalling this module manually too (with `button_immediate_install()`) instead of changing the order of the tests ? :slightly_smiling_face:

Reply:
Good question! While the theme is reinstalled, test_themes is not. Reinstalling test_themes causes a bunch of errors (most likely cause test_themes creates a lot of records). Since test_themes is not a module installed by users, this fix is faster and probably safer too.